infrastructure/storage/src/lmdb_store/store.rs

```diff
@@ -137,6 +137,22 @@ impl LMDBBuilder {
             target: LOG_TARGET,
             "({}) LMDB environment created with a capacity of {} MB.", path, self.db_size_mb
         );
+
+        // TODO: `env.set_mapsize(new_size * 1024 * 1024)` if `usage` approach the `self.db_size_mb` limit
+        let env_info = &env.info();
+        let env_stat = &env.stat();
+        let size_used = *&env_stat.as_ref().unwrap().psize as usize * *&env_info.clone().unwrap().last_pgno;
+        let space_remaining = *&env_info.clone().unwrap().mapsize - size_used;
+        let usage = (*&size_used as f64 / *&env_info.clone().unwrap().mapsize as f64) * 100.0;
+        info!(
```

Comment:
I think this should rather be debug. The user has no control here, as its a constant. So no use printing it out this fine.

Reply:
Agreed

Reply:
Agreed